OracleUnifiedDirectory/CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing
```

Comment:
There should not be a `CONTRIBUTING.md` file in your submission. We already have one in this repo.